servers/audio_server.cpp

```diff
@@ -1079,11 +1088,11 @@ double AudioServer::get_output_latency() const {
 	return AudioDriver::get_singleton()->get_latency();
 }
 
-double AudioServer::get_time_to_next_mix() const {
```

Comment:
Is it really needed to remove the `const`?

Reply:
No, I don't HAVE to remove const on the `AudioServer` but I do have to remove it from `AudioDriver` - I removed it because I didn't find any other example of a `const` `AudioServer` method calling a non-const `AudioDriver` method, however I can put it back.

Reply:
I don't think we have a strong coding standard for these cases. However, since these functions are conceptually constant, let's keep the `const`.

Reply:
Done.